### PR TITLE
Migrate article about Grafana and OpenSearch integration

### PR DESCRIPTION
**Pull request by anelook**
migrated https://help.aiven.io/en/articles/2093033-setting-up-elasticsearch-alerts-in-grafana
The initial article is quite outdated. I also excluded the exact steps of using Grafana. Their interface is changing and also, if we need these instructions they should belong to Grafana section.

### CODE REVIEW

File: docs/products/opensearch/howto/integrate-with-grafana.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Follow :doc:`these instructions </docs/products/grafana/howto/log-in>` to log in into Aiven for Grafana.
+#. In *Configuration menu* select **Data sources**.
+#. Click to **Add data source**.
+#. Find **OpenSearch** in the list and select it. You'll see a panel with list of settings to fill in.
+#. Use your preferred in the *Name* field. You'll use it later for creating dashboards and alerts.
+#. Set *URL* to ``OPENSEARCH_URI``.
+#. In *Auth* section enable **Basic auth** and **With Credentials**.
+#. In *Basic Auth Details* set your ``OPENSEARCH_USER`` and ``OPENSEARCH_PASSWORD``.
+#. Scroll down to *OpenSearch details* and set the index name(for example, ``logs-*``).
```
Comment (ftisiot) [inline docs/products/opensearch/howto/integrate-with-grafana.rst:37]: should this be the index pattern? Since i believe you can use wildcards?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Really good @anelook, Minimal change possibly needed!

File: docs/products/opensearch/howto/integrate-with-grafana.rst (hunk 1 above)
Comment (anelook) [reply docs/products/opensearch/howto/integrate-with-grafana.rst:37]: You're right. They call it index name in Grafana data source. But both work. So I changed that!

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**